service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/C2DocumentBundle.java

```diff
@@ -11,7 +12,10 @@
     private final String nameOfRepresentative;
     private final String pbaNumber;
     private final String clientCode;
-    private final String fileReference;
+
+    @JsonProperty(value = "fileReference")
```

Comment:
I guess we could check with devops if we have any entries like that (probably not, and then don't have JsonProprety), but I am noot massively concerned.